source/common/router/router.h

```diff
@@ -611,6 +613,7 @@ class Filter : Logger::Loggable<Logger::Id::router>,
   bool request_buffer_overflowed_ : 1;
   const bool streaming_shadows_ : 1;
   bool upstream_request_started_ : 1;
+  bool orca_load_report_received_ : 1;
```

Comment:
Please add a short description of the flag. based on:
```
  // Process the load report only once, so if response has report in headers,
  // then don't process it in trailers.
```

Reply:
Done